notifications.py

```diff
@@ -0,0 +1,16 @@
+def median_list(nums):

```

Comment:
Problem statement por favor.

Reply:
The problem statement is provided in commented section. I will mention on file too here after.

https://www.hackerrank.com/challenges/fraudulent-activity-notifications/problem